WebApp-OpenIDConnect-DotNet/Views/Shared/_LoginPartial.cshtml

```diff
@@ -2,6 +2,8 @@
 {
     <ul class="nav navbar-nav navbar-right">
         <li class="navbar-text">Hello @User.Identity.Name!</li>
+        <li><a asp-area="" asp-controller="Session" asp-action="EditProfile">Edit Profile</a></li>
```

Comment:
Normally we make the user name text the edit profile link, like this: https://github.com/dotnet/templating/blob/rel/vs2017/rtw/template_feed/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_LoginPartial.cshtml#L12